Packs/Arcanna/Scripts/ArcannaFeedbackPostProcessingScript/ArcannaFeedbackPostProcessingScript.py

```diff
@@ -0,0 +1,103 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+ARCANNA_AUTO_CLOSED_TICKET_PLAYBOOK_PLACEHOLDER = "Arcanna decision:"
+
+
+def get_value_from_context(key):
+    return demisto.get(demisto.context(), key)
+
+
+def send_arcanna_feedback(close_notes, close_reason, closing_user, event_id, job_id):
+    ret = demisto.executeCommand("arcanna-send-event-feedback", {
+        "job_id": job_id,
+        "event_id": event_id,
+        "label": close_reason,
+        "username": closing_user,
+        "closing_notes": close_notes
+    })
+    return ret
+
+
+def extract_feedback_information():
+    feedback_field = get_value_from_context(key="Arcanna.FeedbackField")
+    if type(feedback_field) == list:
+        feedback_field = feedback_field[0]
+
+    if feedback_field is None or feedback_field == "":
+        raise Exception("Failed to get value for Arcanna closing field")
+
+    # Get Values from incident
+    feedback_field_value = demisto.incident().get(feedback_field, None)
+    if feedback_field_value is None or feedback_field_value == "":
+        # if closing field value is Empty try to get it from Args as a fallback
+        feedback_field_value = demisto.args().get(feedback_field, None)
```

Comment:
Please explain a bit more about that line. Which args should be in the variable `feedback_field`?

Reply:
if the script params contains feedback_field try to read it from there also